src/assets/styles/components/_filters.css

```diff
@@ -0,0 +1,149 @@
+
+[class*="--hide"] {
+    display: none;
+}
+
+.filters {
+    margin-block-end: 1.75rem;
+
+    inclusive-disclosure {
```

Comment:
The `:focus-within` style is not distinguishable from the `:focus` style, see:

Disclosure button has focus:

<img width="382" alt="Type filter expanded, Standards checkbox has focus" src="https://github.com/user-attachments/assets/730cb6f5-e40e-40a7-b0e6-cf7bc1bff884" />

Type button has focus:

<img width="377" alt="Type filter expanded, focus is on Type button" src="https://github.com/user-attachments/assets/143a1e69-694f-4975-b046-8f6164fc13eb" />

Reply:
@uttaraghodke